### ISSUE TITLE
Possible URL bug

### ISSUE THREAD

**Issue by Dextrolain** [xenocrat/chyrp-lite#63]
Hi. 
I noticed a small problem navigating the pages bar on mobile version.
Not sure how to describe it.
You can recreate it this way:
If you create two pages visible in navigation bar, for example "parent1" and "parent2" and one children page ("child1") for "parent1" and navigate to mysite.com/parent1/child1 then open the bottom menu bar and click parent2 (or parent1 or whatever other page you may have there) you'll find yourself on mysite.com/parent1/child1/parent2.

Actually that link would still work and show the actual content of parent2, but further navigation through the pages in the menu would point to 404 page.

Same would be true for other URLs like:
mysite.com/parent1/child1/feed
mysite.com/parent1/child1/index
etc., if "Clean Urls" is enabled

Full size version doesn't have this problem.

Hope I didn't confuse you with this description :)

**Comment by xenocrat** [comment]
Thanks for the report. This is a bug caused by me in _default.twig_ because I used `item.url`, a property that is a synonym for the page's slug, instead of `item.url()`, a method that generates an absolute URL for the page.

It's a bit confusing to have a method and a property share a name, but that's what I inherited from the upstream project. Maybe one day I will remove the `item->url` property from pages and posts and we'll see what breaks. ^_^

Fixed in master now.
